app/src/main/java/org/mozilla/fenix/home/recentsyncedtabs/RecentSyncedTabFeature.kt

```diff
@@ -80,13 +81,17 @@ class RecentSyncedTabFeature(
         lastSyncedTab = syncedTab
     }
 
-    // UI will either not be displayed if not authenticated (DefaultPresenter.start),
-    // or the display state will be tied directly to the success and error cases.
-    override fun stopLoading() = Unit
+    // This is called in success cases as well, but the state should only change if there isn't a
+    // tab displayed. The store's state isn't updated in time to rely on it for this condition,
+    // so local state is used instead.
```

Comment:
```suggestion
    /**
     * This is called in success cases as well, but the state should only change if there isn't a
     * tab displayed. The store's state isn't updated in time to rely on it for this condition,
     * so local state is used instead.
     */
```

nit: could you fix the comment formatting to a doc since you're changing the comment itself. Inner comments use the `//`, on the method should follow the doc style with something like "implementation notes" or "NB" prepended where desired.

Reply:
Sure thing!